### ISSUE TITLE
Clean up legacy in Job class schema

### ISSUE THREAD

**Issue by egede** [ganga-devs/ganga#2110]
The Job class has some very old legacy fields in the [schema](https://github.com/ganga-devs/ganga/blob/45d523f0895c2992a4c9bc5aba1fae6b5a3867f8/ganga/GangaCore/GPIDev/Lib/Job/Job.py#L207). The fields `inputsandbox`, `outputsandbox` and `outputdata` can be removed. 

The rest of the code base should be checked and places where these are used should be removed as it all deals with legacy ways of specifying a job.

The `ForbidLegacyOutput` config option should be removed as well. This last bit is related to #2109.

**Comment by vedanshbhartia** [comment]
Hi, I was looking into this issue. Should the ForbidLegacyInput Option be removed as well as part of this change?

**Comment by egede** [comment]
Yes, it should indeed. Make sure to keep the inputdata field in the schema though.

**Comment by tanuj-rai** [comment]
I am working on this issue, but I would like to know what I should write in place of the field that is being removed. In conditional statements fields are used so how should I remove them?

**Comment by egede** [comment]
> I am working on this issue, but I would like to know what I should write in place of the field that is being removed. In conditional statements fields are used so how should I remove them?

I believe that @vedanshbhartia is already working on this issue. As a reply to your question, the `ForbidLegacyOutput` output option can be regarded as always being `True`, so conditional statements should be removed to reflect this.

**Comment by vedanshbhartia** [comment]
Some components (such as the `preparejob` funtion in `ganga/GangaCore/Lib/LCG/LCG.py`) contain input_sandbox and output_sandbox fields. However, their usage is not wrapped around a check for ForbidLegacy[Input/Output]. Are these variables different from the sandboxes used in Job.py?

**Comment by egede** [comment]
That rather reflects that the complete `LCG` backend should be removed. Might be best to do that in a separate PR first in fact.

**Comment by Ishaan-Chandak** [comment]
Hi @egede,

Can I work on this ? if the issue is still open

**Comment by egede** [comment]
> Hi [@egede](https://github.com/egede),
> 
> Can I work on this ? if the issue is still open

Yes, you are welcome to. Note that the `LCG` backend has been removed, making the changes easier to implement.
